.vib/gitea/goss/gitea.yaml

```diff
@@ -0,0 +1,59 @@
+group:
+  gitea:
+    exists: true 
+user:   
+  gitea:
+    exists: true
+file:
+  /opt/bitnami/gitea:     
+    exists: true     
+    mode: "0775"     
+    owner: gitea     
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/bin/gitea:
+    exists: true
+    mode: "0775"
+    filetype: file
+  /opt/bitnami/gitea/custom:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/data:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/tmp:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/tmp/log:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /bitnami/gitea:
+    exists: true
+    mode: "0775"
+    owner: gitea
+    group: root
+    filetype: directory
+  /opt/bitnami/gitea/custom/conf/app.ini:
+    exists: true
+    mode: "0664"
+    owner: gitea
+    group: root
+    filetype: file
+command:
+  check-app-run:
+    exec: timeout --preserve-status 5 gitea web
+    exit-status: 0
+    stdout:
+      - "Starting new"
```

Comment:
Checking the commands stdout, I think we may need to use a more specific string to check. Maybe something like `/Starting.*web.*server/` could work here, wdyt?